### DISCUSSION TITLE
Is there any way to connect firebase using ipv6?

### DISCUSSION THREAD

**Discussion by Robiullah2244** [invertase/react-native-firebase#6937] [General]
I have implemented Firebase Storage in my React Native project to play audio, and it is functioning properly on all of our devices.

However, during the Apple review process, they reported that they were unable to access the audio from Firebase since they are using the ipv6 network to test the app.

> Guideline 2.1 - Performance - App Completeness
> 
> We continue to find that one or more bugs in your app. Specifically, the activity indicator spun indefinitely when we taped on the speaker button. Please review the details below and complete the next steps.

Please help me out.

**Comment by mikehardy** [comment]
Interesting question! I honestly have no idea - I suggest asking this on stackoverflow with the correct tags, as the official firebase support team responds there (here we are just wrapping their work in the form of their firebase-ios-sdk and firebase-android-sdk libraries)

**Reply by Robiullah2244** [reply]
Thank you so much. I have created a StackOverflow question just now. Here is the link: https://stackoverflow.com/questions/75568893/is-there-any-way-to-connect-firebase-using-ipv6-6937

**Reply by mikehardy** [reply]
Official support already engaged! Fantastic, hopefully we can figure something out. There's always the chance it's react-native-specific somehow but perhaps firebase storage only does ipv4, my guess is Doug (on stackoverflow) will clear that bit up immediately

**Comment by 0x0elliot** [comment]
any updates?
